plugins/profefe.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: profefe
+spec:
+  version: v0.8.0
+  homepage: https://github.com/profefe/kubectl-profefe
```

Comment:
404. we'll need the project to be open source.

Reply:
wrong link :) fixed